File: frontend/lib/model/student.dart
class Student {
  String id;
  String name;
  String address;
  String mobile;
  DateTime dob;
  String gender;

  Student({
    required this.id,
    required this.name,
    required this.address,
    required this.mobile,
    required this.dob,
    required this.gender,
  });

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'address': address,
      'mobile': mobile,
      'dob': dob.toIso8601String(),
      'gender': gender,
    };
  }

  factory Student.fromJson(Map<String, dynamic> json) {
    return Student(
      id: (json['id'] as int).toString(),
      name: json['name'] as String,
      address: json['address'] as String,
      mobile: json['mobile'] as String,
      dob: DateTime.parse(json['dob'] as String).toLocal(),
      gender: json['gender'] as String,
    );
  }
}

File: back-end/src/entity/Student.ts
import { Entity, PrimaryGeneratedColumn, Column } from "typeorm"

@Entity()
export class Student {
    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    name: string;

    @Column()
    age: number;

    @Column()
    dof: string;

    @Column()
    gender: string;

    @Column()
    address: string;

    @Column()
    mobile: string;


}
